antora-playbook.yml

```diff
@@ -0,0 +1,18 @@
+site:
```

Comment:
Can this be moved into `etc`?

Reply:
Sure, I can also move the `package.json` and `package-lock.json` to `etc` if you want?